src/ziggurat/metrics.clj

```diff
@@ -56,19 +56,23 @@
 
 (defn- inc-or-dec-count
   ([sign metric-namespace metric]
-   (inc-or-dec-count sign metric-namespace metric nil))
-  ([sign metric-namespaces metric additional-tags]
-   (let [metric-namespace (get-metric-namespaces metric-namespaces)
-         meter            ^Meter (mk-meter metric-namespace metric (remove-topic-tag-for-old-namespace additional-tags metric-namespaces))]
-     (.mark meter (sign 1)))))
+   (inc-or-dec-count sign {:metric-namespace metric-namespace :metric metric :n 1 :additional-tags {}}))
+  ([sign metric-namespace metric n]
+   (inc-or-dec-count sign {:metric-namespace metric-namespace :metric metric :n n :additional-tags {}}))
+  ([sign metric-namespace metric n additional-tags]
+   (inc-or-dec-count sign {:metric-namespace metric-namespace :metric metric :n n :additional-tags additional-tags}))
+  ([sign {:keys [metric-namespace metric n additional-tags]}]
```

Comment:
this n and additional-tags should be optional, that was the reason why I suggested using a map here. So, if n is nil, we should default it to 1 and if additional-tags in nil we should default it to {}